logistics_project/apps/malawi/warehouse/report_utils.py

```diff
@@ -214,17 +214,16 @@ def get_reporting_rates_chart(location, start, end, is_facility=False):
     })
     return report_chart
 
-def get_multiple_reporting_rates_chart(supply_points, date):
+def get_multiple_reporting_rates_chart(supply_points, date, is_facility=False):
     """
     Reporting rates chart for multiple facilities, for a single 
     month.
     """
     report_chart = _reporting_rate_chart_defaults()
     data = defaultdict(lambda: defaultdict(lambda: 0)) # turtles!
-    rrs = ReportingRate.objects.filter(supply_point__in=supply_points, date=date)
+    rrs = ReportingRate.objects.filter(supply_point__in=supply_points, date=date, is_facility=is_facility)
     for rr in rrs:
         sp = rr.supply_point
-        rr = ReportingRate.objects.get(supply_point=sp, date=date)
```

Comment:
as far as i can tell this line is redundant. i also opened the dashboard with and without this line and it looked the same